libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -46,9 +59,30 @@ export class BookSearchComponent implements OnInit,OnDestroy {
   }
 
   addBookToReadingList(book: Book) {
+    
+    this.item ={
+      bookId:'',
+      title:'',
+      authors:[],
+      description:''
+  
+    };;
+    this.item.bookId=book.id;
     this.store.dispatch(addToReadingList({ book }));
+    this.openSnackBar("Book Added!!","Undo");
+   
+  }
+  onChanges(){
+    this.searchForm.get('term').valueChanges.pipe(
```

Comment:
See my comment in the other PR https://github.com/mahender-surkanti/okreads-puzzle/pull/3.